### ISSUE TITLE
Deploy from IBM Container Registry to Kubernetes

### ISSUE THREAD

**Issue by christophernixon** [christophernixon/DevOps-Pipeline-sweng#44]
Dependent on Issue #38. Once we have our Docker Image in our Container Registry in IBM cloud, we need to deploy this to Kubernetes, so that we have a publicly exposed places to view it. 
> At the moment whenever we deploy to heroku we can view it at https://react-app-develop.herokuapp.com/

There are fairly details steps on how this would be achieved for the first time from this IBM repo: https://github.com/IBM/deploy-react-kubernetes/. Another interesting and related article can be found [here](https://medium.com/@rizchelwood/my-journey-to-deploying-my-first-react-application-with-kubernetes-95e6d25f434d)

The first task would be:
- [x] Deploying our Image to Kubernetes for the first time, probably by following the above tutorial.

Secondly:
- [x] Automate this process using Travis CI.
This would involve adding more commands to the deploy script from Issue #38, probably `kubectl` would have to be installed and then the kubectl commands could be run. This may be too much for one Issue so it might be better to split it into smaller tasks.
